README.md

```diff
@@ -38,7 +38,7 @@ Sometimes you'll have an activity that should handle several kinds of deep links
 `@DeepLinks` annotation to register multiple deep links on an activity:
 
 ```java
-@DeepLinks({"example.com/deepLink/{id}", "example.com/anotherDeepLink"})
+@DeepLinks({"foo://example.com/deepLink/{id}", "foo://example.com/anotherDeepLink"})
```

Comment:
Compilation will fail if the URI is invalid. It is now being checked by `DeepLinkAnnotatedElement`